scipy/optimize/tests/test_optimize.py

```diff
@@ -2450,6 +2451,11 @@ def test_equal_bounds(method, kwds, bound_type, constraints, callback):
     gh12502 - Divide by zero in Jacobian numerical differentiation when
     equality bounds constraints are used
     """
+    # GH-15051; slightly more skips than necessary; hopefully fixed by GH-14882
+    if (platform.machine() == 'aarch64' and method == "TNC"
+            and kwds["jac"] is False and callback is not None):
+        pytest.skip('Tolerance violation on aarch')
```

Comment:
sounds like Ralf & Andy were "ok" with this skip, which would then be reverted when Andy's Cythonization PR goes in